modules/javafx.controls/src/test/java/test/javafx/scene/control/MenuItemTest.java

```diff
@@ -275,19 +268,13 @@
     @Test public void getUnspecifiedGraphicProperty1() {
         MenuItem mi2 = new MenuItem();
         assertNotNull(mi2.graphicProperty());
+        assertNull(mi2.getGraphic());
     }
 
     @Test public void getUnspecifiedGraphicProperty2() {
         MenuItem mi2 = new MenuItem("",null);
         assertNotNull(mi2.graphicProperty());
-    }
-
-    @Ignore // Again, calling graphicPropery() is not ensuring a non null graphic
-    // node.
-    @Test public void unsetGraphicButNotNull() {
```

Comment:
Test was incorrect. The value of the 'graphic' property is not set and asserting on not null is incorrect.